.github/workflows/pypi.yml

```
@@ -17,9 +17,34 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
```

Make zizmor happy about default permissions

```suggestion
permissions: {}

jobs:
```